frontend/src/flashcard.tsx

```diff
@@ -89,14 +89,15 @@ const QuestionDisplay: React.FC<QuestionDisplayProps> = ({
   tileStyle,
   theme,
 }) => {
-  console.log("tileStyle", { tileStyle, fontStyle });
+  const isSmallScreen = useIsSmallScreen();
+
   switch (tileStyle) {
     case TileStyle.MatchDisplay: {
       return (
         <TiledText
-          size="xxl"
-          h={rem(40)}
-          w={rem(40)}
+          size={isSmallScreen ? rem(22) : "xxl"}
+          h={rem(isSmallScreen ? 32 : 40)}
```

Comment:
I imagine there's a way to get it to work with pure CSS/mantine configs but was struggling to do so, JS seems like the simplest approach. the `size` for text doesn't support responsive config so it's a decent amount of ad-hoc CSS to add that